benchmarks/benchmarks/bench_function_base.py

```diff
@@ -95,47 +95,159 @@ def time_select_larger(self):
         np.select(self.cond_large, ([self.d, self.e] * 10))
 
 
+class SortGenerator(object):
+    # The size of the unsorted area in the "random unsorted area"
+    # benchmarks
+    AREA_SIZE = 10
+    # The size of the "partially ordered" sub-arrays
+    BUBBLE_SIZE = 10
+
+    @staticmethod
+    def _random(size, dtype):
+        arr = np.arange(size, dtype=dtype)
+        np.random.shuffle(arr)
+        return arr
+    
+    @staticmethod
+    def _ordered(size, dtype):
+        return np.arange(size, dtype=dtype)
+
+    @staticmethod
+    def _reversed(size, dtype):
+        return np.arange(size-1, -1, -1, dtype=dtype)
+
+    @staticmethod
+    def _uniform(size, dtype):
+        return np.ones(size, dtype=dtype)
+
+    @staticmethod
+    def _type_swapped_pair(size, dtype, swap_frac):
```

Comment:
I don't understand what this function name means. "type-swapped pair"? "type: swapped pair"?
I think the `_type_` prefix is throwing me off on all of these. Perhaps add a comment above them.

Reply:
Removed, and added a bit of docs. Should be more self explanatory now.